src/main/resources/pirk.properties-repo

```diff
@@ -0,0 +1,243 @@
+###############################################################################
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+###############################################################################
+
+##
+## Default System Configuration Properties
+##
+
+# Name of log4j properties file (relative to current folder)
+log4jPropertiesFile=log4j2.properties
+
+#Name of the directory holding the local properties files
+#All property files must end in .properties
+local.pirk.properties.dir=/root/properties
+
+##
+##Spark path for SparkLauncher
+##
+spark.home = /usr
+
+##
+## Data schema properties
+##
+## Each data schema should be specified in an xml file of the form; 
+## 		all items are treated in a case insensitive manner:
```

Comment:
No longer true that they are all case insensitive.
Consider referring to Pirk's website rather than recreate schema definition?